ArmVirtPkg/ArmVirt.dsc.inc

```diff
@@ -443,3 +439,56 @@
     <LibraryClasses>
       NULL|EmbeddedPkg/Library/PlatformHasAcpiLib/PlatformHasAcpiLib.inf
   }
+
+!if ($(NETWORK_TLS_ENABLE) == TRUE || $(SECURE_BOOT_ENABLE) == TRUE || $(TPM2_ENABLE) == TRUE) && $(ARCH) == AARCH64

```

Comment:
nit: I can’t remember if the arch syntax works here, too, Components.AARCH64 I think does work, LibraryClasses.AARCH64.* I think is supposed to work, I’m not sure if it does.